packages/augur-sdk/src/utils.ts

```diff
@@ -149,17 +149,17 @@ export function calculatePayoutNumeratorsValue(
   } else {
     switch(marketType) {
       case MarketTypeName.Categorical:
-        if (!isWellFormedCategorical(payout)) return { malformed: true };
+        if (!isWellFormedCategorical(payout)) return { outcome: '0', malformed: true };
```

Comment:
I'm finally catching up to why outcome wasn't included. 

We need outcome set to 0 when outcome is invalid

lets not include outcome when malformed is true. We don't want to tell caller incorrect information. 

also make outcome nullable so change to `outcome?`

Reply:
Ah. Yeah I can do that.